### PR TITLE
Sunny weather

### PR DESCRIPTION
**Pull request by MaxRobinsonTheGreat**
- Weather commands now have `sunny` weather available, which basically sets the skysphere back to its default values and removes any fog or particle systems. BYU-PCCL/holodeck#376
- Set weather commands no longer interfere with each other, so `set_weather("rain")` can be called after `set_weather("cloudy")` without problems.
- Editor now crashes when an environment does not have a custom command. This happens when a weather command is called and the skysphere, skylight, or directional light are missing.
- Error message: `World cannot execute given command: xxxx. Different holodeck environments have different available commands. Read the docs to ensure you are using an environment with the desired functionality.`
- Fixed BYU-PCCL/holodeck#384

### CODE REVIEW

File: Source/Holodeck/HolodeckCore/Private/HolodeckGameMode.cpp [hunk 1]
```diff
@@ -63,3 +63,7 @@ void AHolodeckGameMode::RegisterSettings() {
 		UE_LOG(LogHolodeck, Log, TEXT("Reset signal registered"));
 	}
 }
+
+void AHolodeckGameMode::LogFatalMessage(const FString& Message) {
+	UE_LOG(LogHolodeck, Fatal, TEXT("%s"), *Message);
+}
```
Comment (MaxRobinsonTheGreat) [inline Source/Holodeck/HolodeckCore/Private/HolodeckGameMode.cpp:69]: This function is to allow blueprints to log fatal messages. I could not find a way to do this from blueprints directly.

**Comment by vinhowe** [comment]
@MaxRobinsonTheGreat @jaydenmilne is this ready to merge? Is merging this all we need to close https://github.com/BYU-PCCL/holodeck/issues/376?

**Review by jaydenmilne — APPROVED** [review]
I'm trusting your blueprint-foo

**Comment by jaydenmilne** [comment]
Once the Python side is in and a note in the changelog we can close
BYU-PCCL/holodeck#376. <https://github.com/BYU-PCCL/holodeck/issues/376>
It's not over till its over ;)

**Pull request merged**